### PR TITLE
[REVIEW] Announcement for presentations and speakers

### PR DESCRIPTION
**Pull request by thatdocslady**
Need more info:
- List of speakers and presentations (is there an automagic way of producing this?)
- New sponsors to announce since the last announcement?
- AOB?

Cheers!

### CODE REVIEW

**Comment by plaindocs** [comment]
Looks ok to me. @ericholscher can automagically populate the list, but is presumably waiting for me to finalize the confirmations.

We need to add Travis and Netconomy? To the email + site and whatever. Adrienne posted the logos in the EU slack, but I've not got round to adding them yet.

**Comment by plaindocs** [comment]
Cherry-pick e29db95 if you want the sponsors.

**Comment by ericholscher** [comment]
Updated the speaker list.

File: docs/conf/eu/2016/news/announce-presentations.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* Rory Tanner - `Information micro-architecture: grammar, syntax, and cognitive rhetoric <http://www.writethedocs.org/conf/eu/2016/speakers/#speaker-rory-tanner>`_
+* joan wendt - `Operations Technical Writing for Data Centers <http://www.writethedocs.org/conf/eu/2016/speakers/#speaker-joan-wendt>`_
+* Sarah Karp - `Watch that tone! Creating an information experience in the Atlassian voice <http://www.writethedocs.org/conf/eu/2016/speakers/#speaker-sarah-karp>`_
+* Rachel Whitton - `Delivering High-Velocity Docs that Keep Pace with Rapid Release Cycles <http://www.writethedocs.org/conf/eu/2016/speakers/#speaker-rachel-whitton>`_
+* Margaret Eker, Jennifer Rondeau - `Docs as Code: The Missing Manual <http://www.writethedocs.org/conf/eu/2016/speakers/#speaker-margaret-eker>`_
+
+Tickets
+-------
+
+Tickets are going fast and we expect the conference to sell out again, so get `your tickets now <https://ti.to/writethedocs/write-the-docs-eu-2016/>`_.
```
Comment (ericholscher) [inline docs/conf/eu/2016/news/announce-presentations.rst:37]: Should probably include more in this section, or remove it and just stick it at the top.

**Comment by plaindocs** [comment]
DONE - Oops, we need to add Erik, and should probably fix the capitalization of joan as well.

File: docs/conf/eu/2016/news/announce-presentations.rst (hunk 1 above)
Comment (thatdocslady) [inline docs/conf/eu/2016/news/announce-presentations.rst:37]: @ericholscher I'm having a bit of a crazy day at work, so for the sake of saving time do you mind moving the text to the top (after the intro para and before the speaker list)? thanks!!

Comment (plaindocs) [inline docs/conf/eu/2016/news/announce-presentations.rst:37]: DONE

**Pull request merged**